content/topics/java-specific/spring-boot-rest/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Spring Boot REST tutorial
+ready: true
+---
+
+This document highlights the way in which the various types of endpoints are created in spring api's
```

Comment:
Can you add background about REST vs SOAP for example, and what is REST and why its important